### ISSUE TITLE
Exploring in new update causing massive lag as well as (and maybe caused by) a 700MB log file

### ISSUE THREAD

**Issue by mrbaggins** [Beyond-Reality/Beyond-Reality-Divergence#83]
So after the latest update, running around exploring causes entities to lag horribly, and you can easily outrun world gen. During world gen, the log is being FILLED with the following, over and over again: (Note each line is different, look at far right. This is one "chunk" of the log, this chunk is repeated over and over)

```
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:201]: Hard Ore block, tile.ore_diamond has no flower data.  This is a bug!
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:203]: Listing all mapped blocks:
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_iron:-1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_gold:-1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.beyondrealitycore:oreCopper:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_diamond:-1 true
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.beyondrealitycore:oreTin:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_redstone:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_redstone:1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.beyondrealitycore:oreLead:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_tin:-1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.beyondrealitycore:oreSilver:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_copper:-1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     copper_ore:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.beyondrealitycore:oreNickel:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_lead:-1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.beyondrealitycore:oreAluminium:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_uranium:-1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_silver:-1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.beyondrealitycore:oreCadmium:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     thorium_ore:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_nickel:-1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ore_aluminum:-1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     uranium_ore:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.beyondrealitycore:oreUranium:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.beyondrealitycore:oreZinc:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.tconstruct.stoneore:4 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.tconstruct.stoneore:3 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.tconstruct.stoneore:5 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.tconstruct.gravelore:3 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.tconstruct.gravelore:2 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.tconstruct.gravelore:4 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.dummy_tin:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.dummy_copper:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.dummy_lead:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.dummy_uranium:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.dummy_silver:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.dummy_nickel:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.dummy_aluminum:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.thermalfoundation.ore:1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.thermalfoundation.ore:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.thermalfoundation.ore:3 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.thermalfoundation.ore:2 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.thermalfoundation.ore:4 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.for.resources:2 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.for.resources:1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ImmersiveEngineering.ore:0 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ImmersiveEngineering.ore:2 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ImmersiveEngineering.ore:3 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ImmersiveEngineering.ore:4 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     tile.ImmersiveEngineering.ore:1 false
[15:43:27] [Server thread/INFO] [STDOUT/]: [com.draco18s.ores.block.ores.BlockHardOreBase:func_149726_b:206]:     zinc_ore:0 false
```

Partially unrelated, on game start up, I also got thousands of entries of 

```
[13:09:34] [Server thread/DEBUG] [Matter Overdrive/mo]: There was a problem getting a Matter Entry.
```

And it took several "passes" that look like this. These are interspersedwithin the tens of thousands of the above line.:

```
[13:09:31] [Server thread/INFO] [Matter Overdrive/mo]: Matter Recipe Calculation for pass 5 complete. Took 7389 milliseconds. Registered 13 recipes
[13:09:31] [Server thread/INFO] [Matter Overdrive/mo]: Matter Recipe Calculation Started for 21802 recipes at pass 6, with 2673 matter entries
```

And another odd entry showing up thousands of times:

```
[13:09:54] [Server thread/WARN]: Skipping BlockEntity with id sfartifacts_TileEntityOreAsgardite
```

Long story short, I have a 700MB log file from 2 hours of game play.

**Comment by Dominance** [comment]
Known issue looking into it. Thanks a ton for reporting.

**Comment by mrbaggins** [comment]
Looks like that's the first one, and probably the most important one, but the matter overdrive stuff isn't ideal either. Hope you and draco work it out!

**Comment by Dominance** [comment]
The matter overdrive one is not a issue. That happens on game load and has to be done. Really no different than how EE3 or clones handles it.

**Comment by Dominance** [comment]
Does the SFArtifact clear itself out or is it repeating spam?

**Comment by mrbaggins** [comment]
I get the passes are, but in doing them, I got tens of thousands of the "Problem getting a matter entry" lines. Is that normal? It takes a full minute just spamming the log file with that.

The artifact logs are in the latest.log instead of fml-client, and they repeat for the entire duration, however, not nearly as badly as the ore flowers.

**Comment by Dominance** [comment]
Can confirm the SF artifact spam only happens in existing worlds/chunks I think its just clearing itself out as the ores where changed to help performance over all. So only issue awaiting this would be the flower spam which is still happening to some.

**Comment by Dominance** [comment]
Should be good to go in upcoming release thanks for the detailed reports!
